### PR TITLE
Add Japanese accessibility docs for ColorPicker

### LINKED ISSUE #1543: Add accessibility section for `color-picker`
### Subject

accessibility section for `color-picker`

#### this issue include file below
 - use-color-slider.ts
 - use-saturation-slider.ts
#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://www.radix-ui.com/primitives/docs/components/accordion#accessibility
https://mantine.dev/core/color-picker/#accessibility

#### Include the following content
* keboard interaction
* aria, tabindex property are supported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #1543

## Description

Add Japanese accessibility docs for ColorPicker.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: docs/contents/components/forms/color-picker/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+|                                         | `aria-valuemin`        | `0`の値を設定します。                                                                                        |
+|                                         | `aria-valuemax`        | `360`の値を設定します。                                                                                      |
+|                                         | `aria-valuenow`        | 現在の値を設定します。                                                                                       |
+|                                         | `aria-valuetext`       | `18°, Red`のように、現在の値を設定します。                                                                   |
+| `AlphaSliderThumb` <Tag>内部</Tag>      | `role="slider"`        | スライダーであることを示します。                                                                             |
+|                                         | `aria-label`           | `"Slider thumb"`を設定します。                                                                               |
+|                                         | `aria-valuemin`        | `0`の値を設定します。                                                                                        |
```
Comment (kaehehehe) [inline docs/contents/components/forms/color-picker/accessibility.ja.mdx:68]:
```suggestion
|                                         | `aria-valuemin`        | `0`の値を設定します。 デフォルトは`0`です。                                                                                      |
```

File: docs/contents/components/forms/color-picker/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|                                         | `aria-valuemax`        | `1`の値を設定します。                                                                                        |
```
Comment (kaehehehe) [inline docs/contents/components/forms/color-picker/accessibility.ja.mdx:69]:
```suggestion
|                                         | `aria-valuemax`        | `1`の値を設定します。　デフォルトは`1`です。                                                                                        |
```

**Review by kaehehehe — CHANGES_REQUESTED** [review]

File: docs/contents/components/forms/color-picker/accessibility.ja.mdx (hunk 1 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/color-picker/accessibility.ja.mdx:68]: Don't change.

File: docs/contents/components/forms/color-picker/accessibility.ja.mdx (hunk 2 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/color-picker/accessibility.ja.mdx:69]: Don't change.

**Review by kaehehehe — APPROVED** [review]

**Pull request merged**